### ISSUE TITLE
[BUG] command /f getvault replace item you are holding with vault chest

### ISSUE THREAD

**Issue by TacoSeller420** [EarthMC/Issue-Tracker#744]
The bug is on the server EarthMc factions

Basically, when I did the command /f getvault, it switched my sword which was basically full enchanted sword missing only looting to a vault chest... the worse is that it took me some time to realise what happen... but yeah. 
Basically switched items

**To Reproduce**
hold X item in hand
do /f getvault
idk if it affect the bug or not, but if someone leave at the same time and create a little lag, maybe it influence the bug, idk this thing.
item is replaced with chest vault
**Expected behavior**
either that vault are disabled or that its a "cloud" type vault

basically this was my last screenshot i took of my op sword.
![opsword](https://user-images.githubusercontent.com/54733311/64059350-cefa1f00-cb88-11e9-8e8f-edc1a7c12a08.png)
.
later on i do the thing and bam: this happen
![replacedwithvault](https://user-images.githubusercontent.com/54733311/64059352-e33e1c00-cb88-11e9-9e87-00dc7a88d205.png)

I also hate my life

**Comment by KarlOfDuty** [comment]
The factions plugin does this, it would have to be reported to them to fix it.
